larray/example.py

```diff
@@ -0,0 +1,54 @@
+import os
+import larray as la
+
+EXAMPLE_FILES_DIR = os.path.dirname(__file__) + '/tests/data/'
+AVAILABLE_EXAMPLE_DATA = {
+    'demography' : EXAMPLE_FILES_DIR + 'data.h5'
+}
+
+def load_example_data(example_data=None):
```

Comment:
I am unsure having a default value is a good idea? Wouldn't "name" be a better argument name?